src/Containers/FeaturedProfile.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import { flex } from '../Utils/Mixins';
+
+export interface IFeaturedProfileProps {
+    background: string;
+    profile: IProfile;
+}
+
+export interface IProfile {
+    image: string;
+    initials: string;
+    id: number;
+}
+
+export const IconCircle: React.FC = ({ children }) => {
+    return <CircleImage background="gray">{children}</CircleImage>;
+};
+
+export const FeaturedProfile: React.FC<IFeaturedProfileProps> = ({
+    children,
+    background,
+    profile,
+}) => {
+    return (
+        <CircleImage background={background} key={profile.id}>
+            {children}
+        </CircleImage>
+    );
```

Comment:
I don't see the need for two seperate components here, they perform the same function. Props can determine what to display.